static/app/views/replays/detail/network/networkRequestDetails.tsx

```diff
@@ -0,0 +1,197 @@
+import {Fragment, MouseEvent, useCallback, useMemo} from 'react';
+import styled from '@emotion/styled';
+import queryString from 'query-string';
+
+import {Button} from 'sentry/components/button';
+import JSONBlock from 'sentry/components/jsonBlock';
+import Stacked from 'sentry/components/replays/breadcrumbs/stacked';
+import {IconClose} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {useResizableDrawer} from 'sentry/utils/useResizableDrawer';
+import useUrlParams from 'sentry/utils/useUrlParams';
+import SplitDivider from 'sentry/views/replays/detail/layout/splitDivider';
+import NetworkRequestTabs, {
+  TabKey,
+} from 'sentry/views/replays/detail/network/networkRequestTabs';
+import type {NetworkSpan} from 'sentry/views/replays/types';
+
+type Props = {
+  items: NetworkSpan[];
+  initialHeight?: number;
+};
+
+function NetworkRequestDetails({initialHeight = 100, items}: Props) {
+  const {getParamValue: getDetailRow, setParamValue: setDetailRow} = useUrlParams(
+    'n_detail_row',
+    ''
+  );
+  const {getParamValue: getDetailTab} = useUrlParams('n_details_tab', '');
+  const itemIndex = getDetailRow();
+
+  const item = itemIndex ? (items[itemIndex] as NetworkSpan) : null;
+
+  // TODO(replay): the `useResizableDrawer` seems to measure mouse position in relation
+  // to the window with event.clientX and event.clientY
+  // We should be able to pass in another frame of reference so mouse movement
+  // is relative to some container instead.
+  const {
+    isHeld,
+    onDoubleClick,
+    onMouseDown,
+    size: containerSize,
+  } = useResizableDrawer({
+    direction: 'up',
+    initialSize: initialHeight,
+    min: 0,
+    onResize: () => {},
+  });
+
+  const onClose = useCallback(
+    (e: MouseEvent) => {
+      e.preventDefault();
+      setDetailRow('');
+    },
+    [setDetailRow]
+  );
+
+  const data = useMemo(() => getData(item), [item]);
+  if (!data) {
+    return null;
+  }
+
+  const visibleTab = getDetailTab();
+  const tabSections = data[visibleTab];
+  return (
+    <Fragment>
+      <StyledStacked>
+        <StyledNetworkRequestTabs underlined={false} />
+        <StyledSplitDivider
+          isHeld={isHeld}
+          onDoubleClick={onDoubleClick}
+          onMouseDown={onMouseDown}
+          slideDirection="updown"
+        />
+        <CloseButtonWrapper>
+          <Button
+            aria-label={t('Hide request details')}
+            borderless
+            icon={<IconClose isCircled size="sm" color="subText" />}
+            onClick={onClose}
+            size="zero"
+          />
+        </CloseButtonWrapper>
+      </StyledStacked>
+      <SectionList height={containerSize}>
+        {Object.entries(tabSections).map(([label, sectionData]) => (
```

Comment:
I don't know how, but I got into a state where `tabSections` was undefined and it would crash the network section on me.

Repro: https://sentry-i3tuk8psi.sentry.dev/replays/billy-test:cb8c41dbd430498896bc155041496de0/?f_n_type=fetch&n_detail_row=54&query=&referrer=%2Freplays%2F&statsPeriod=14d&t_main=network&yAxis=count%28%29

Reply:
sorted it out. missing default in the hook.